fwoa-tools/src/migrationExport.ts

```diff
@@ -108,47 +112,51 @@ export async function runScript(smartClient: boolean, dryRun: boolean, since: st
   await checkConfiguration(logs, smartClient ? 'Smart' : 'Cognito');
   logs.write('Verification complete');
   if (!dryRun) {
-      const response = await startExport(since, smartClient, snapshotExists, snapshotLocation);
-      logs.write('Export completed. Start sorting objects.');
+    const response = await startExport(since, smartClient, snapshotExists, snapshotLocation);
+    logs.write('Export completed. Start sorting objects.');
 
-      let tenantPrefix = '';
-      if (process.env.MIGRATION_TENANT_ID) {
-        tenantPrefix = `${process.env.MIGRATION_TENANT_ID}/`;
-      }
-      const names = await generateStateFile(EXPORT_BUCKET_NAME!, `${tenantPrefix}${response.jobId}/`);
-      output = {
-        jobId: output.jobId,
-        file_names: names.file_names
-      };
-      // eslint-disable-next-line security/detect-non-literal-fs-filename
-      writeFileSync(`./${EXPORT_STATE_FILE_NAME}`, JSON.stringify(output));
-      logs.write(`${new Date().toISOString()}: Finished sorting export objects into folders.\n`);
+    let tenantPrefix = '';
+    if (process.env.MIGRATION_TENANT_ID) {
+      tenantPrefix = `${process.env.MIGRATION_TENANT_ID}/`;
+    }
+    const names = await generateStateFile(EXPORT_BUCKET_NAME!, `${tenantPrefix}${response.jobId}/`);
+    output = {
+      jobId: output.jobId,
+      file_names: names.file_names
+    };
+    // eslint-disable-next-line security/detect-non-literal-fs-filename
+    writeFileSync(`./${EXPORT_STATE_FILE_NAME}`, JSON.stringify(output));
+    logs.write(`${new Date().toISOString()}: Finished sorting export objects into folders.\n`);
   }
 }
 
-export function buildRunScriptParams(): {smartClient: boolean, dryRun: boolean, since: string,
-  snapshotExists: boolean, snapshotLocation: string} {
+export function buildRunScriptParams(): {
+  smartClient: boolean;
+  dryRun: boolean;
+  since: string;
+  snapshotExists: boolean;
+  snapshotLocation: string;
+} {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const argv: any = parseCmdOptions();
   const smartClient: boolean = argv.smart;
   const dryRun: boolean = argv.dryRun;
   const since: string = argv.since;
   const snapshotExists: boolean = argv.snapshotExists;
   const snapshotLocation: string = argv.snapshotLocation;
-  return {smartClient, dryRun, since, snapshotExists, snapshotLocation}
+  return { smartClient, dryRun, since, snapshotExists, snapshotLocation };
 }
 
 /* istanbul ignore next */
 (async () => {
   // Don't runScript when code is being imported for unit tests
-  if (!process.argv.includes('test')) {
-    const {smartClient, dryRun, since, snapshotExists, snapshotLocation} = buildRunScriptParams();
+  if (!process.env.UNIT_TEST) {
```

Comment:
All in this file is formatting change except for here. String `test` is not in `process.argv` when there're more than one unit test.